tools/apiview/parsers/rust-api-parser/src/main.ts

```diff
@@ -46,6 +46,7 @@ function addSectionHeader(codeFile: CodeFile, headerText: string): void {
         Value: `/* ${headerText} */`,
         NavigateToId: `header-${headerText}`,
         NavigationDisplayName: `/* ${headerText} */`,
+        RenderClasses: ["namespace"],
```

Comment:
Why is punctuation set to render as a namespace?

Reply:
It was intentional.
![image](https://github.com/user-attachments/assets/f7e17ccb-d2f3-446c-81fe-d098f8bf1ee4)

Since the rendered output is too long. I wanted to show quick links to the re-exported items in particular.
It is merely a delimiter of the whole output.
I'll keep that for now, I'll change if we re-consider for the next iteration.